# Accumulated metagenomic studies reveal recent migration, whole genome evolution, and taxonomic incompleteness of orthomyxoviruses

Gytis Dudas <sup>a</sup> and Joshua Batson <sup>b</sup>

<sup>a</sup>Institute of Biotechnology, Life Sciences Centre at Vilnius University, Vilnius, Lithuania
<sup>b</sup>Chan Zuckerberg Biohub, San Francisco CA, USA

### Abstract

Metagenomic studies have uncovered an abundance of novel viruses by looking be- yond hosts of obvious public health or economic interest. The discovery of conserved genes in viruses infecting geographically and phylogenetically diverse hosts has provided important evolutionary context for human and animal pathogens. However, the resulting viral genomes are often incomplete, and analyses largely characterize the distribution of viruses over their dynamics. Here, we show how the accumulated data of metagenomic studies can be integrated to reveal geographic and evolutionary dynamics in a case study of Orthomyxoviridae, the family of RNA viruses containing influenza. First, we use se- quences of the orthomyxovirus Wuhan mosquito virus 6 to track the global migrations of its host. We then look at overall orthomyxovirus genome evolution, finding signifi- cant gene gain and loss across the family, especially in the surface proteins responsible for cell and host tropism. We find that the surface protein of Wuhan mosquito virus 6 exhibits accelerated non-synonymous evolution suggestive of antigenic evolution, and an entire quaranjavirus group bearing highly diverged surface proteins. Finally we quan- tify the progress of orthomyxovirus discovery and forecast that many highly diverged Orthomyxoviridae remain to be found. We argue that continued metagenomic studies will be fruitful for understanding the dynamics, evolution, ecology of viruses and their hosts, regardless of whether novel species are actually identified or not, as long as study designs allowing for the resolution of complete viral genomes are employed.
